docusaurus/docs/quickstarts/network/local-no-docker.md

```diff
@@ -95,7 +95,7 @@ Token: 40d2d280-a633-46c9-8499-ab2e005dd222
 Saving identity 'default' to ${HOME}/.ziti/quickstart/My-Mac-mini.local.domain/ziti-cli.json
 ```
 
-Congratulations! You can now use the `ziti` CLI to interact with Ziti! ([Ziti CLI Help Here](/docs/manage/cli)) The 
+You can now use the `ziti` CLI to interact with Ziti! ([Ziti CLI Help Here](/operations/cli-basics)). The 

```

Comment:
after much discussion, we agreed to break this into fragments and reference the fragments when / where needed